README.md

```diff
@@ -77,3 +77,15 @@ Ensure the following prerequisites are met:
 1. Copy the files in the ansible folder into the control node
 2. Run the ansible playbook by running `ansible-playbook playbook.yml -i inventory.ini`
 3. Go to http://18.134.222.201:5000/ to see the app
+
+# Running in Docker
+
+To run the app in docker you need docker desktop installed.
+
+Development app run:
+1. `docker build --target development --tag todo-app:dev  .`
+2. `docker run --env-file .env --publish 5000:5000 --mount type=bind,source="$(pwd)/todo_app",target=/todo_app todo-app:dev`
```

Comment:
If you look at your Dockerfile, you actually create a root level folder at `/todo_app` and then copy the `todo_app` folder into that, so that we end up with the source code at `/todo_app/todo_app`
I do like having a folder above the source code, but to make this less confusing maybe using a different folder name would be better, e.g. `/app/todo_app` or similar

But for here at least, this command needs:
```suggestion
2. `docker run --env-file .env --publish 5000:5000 --mount type=bind,source="$(pwd)/todo_app",target=/todo_app/todo_app todo-app:dev`
```